daemons/controld/controld_te_actions.c

```diff
@@ -115,9 +115,11 @@ te_crm_command(crm_graph_t * graph, crm_action_t * action)
         }
     }
 
-    CRM_CHECK(on_node != NULL && strlen(on_node) != 0,
-              crm_err("Corrupted command (id=%s) %s: no node", crm_str(id), crm_str(task));
-              return FALSE);
+    if (pcmk__str_empty(on_node)) {
+        crm_err("Corrupted command (id=%s) %s: no node",
+                pcmk__s(id, "<null>"), pcmk__s(task, "without task"));
+        return FALSE;
```

Comment:
we're using no/without/unspecified here. if it doesn't fold into a sentence we might stick with one - e.g. unspecified.

Reply:
I think it depends on the sentence structure, what seems most natural. In this particular case, we can again improve it by testing id/task earlier since they must be present.